app/javascript/scavenger_hunt/scenes/PlayArena/PlayArena.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { Progress } from '../../components/Progress'
+import { Navigation } from './Navigation';
+import { MainContent } from './MainContent';
+import './PlayArena.scss'
+
+const PlayArena = () => {
+  return (
+    <div className="main">        
+      <Progress />
```

Comment:
if progress is moved off the front page then should we have a continue/get started button on the front page or should we also have a progress bar on the front page?

Reply:
I think we can remove it from there., have it in play area

Reply:
ok, worst case we can move it back